src/olympia/versions/views.py

```diff
@@ -95,27 +95,40 @@ def download_file(request, file_id, type=None, file_=None, addon=None):
     if not addon:
         addon = get_object_or_404(Addon.with_unlisted,
                                   pk=file_.version.addon_id)
+    channel = file_.version.channel
+    is_editor = (acl.check_addons_reviewer(request)
```

Comment:
Please make that a function or something to avoid doing it for requests that don't need to, since it has a perf cost.